lib/ansible/galaxy/api.py

```diff
@@ -54,7 +48,12 @@ def should_retry_error(exception):
     # Note: cloud.redhat.com masks rate limit errors with 403 (Forbidden) error codes.
     # Since 403 could reflect the actual problem (such as an expired token), we should
     # not retry by default.
-    if isinstance(exception, GalaxyError) and exception.http_code in RETRY_HTTP_ERROR_CODES:
+    try:
+        retry_codes = [int(code) for code in C.GALAXY_RETRY_HTTP_ERROR_CODES]
+    except ValueError as e:
+        raise AnsibleError("Invalid value for HTTP retry code: %s. Only integer values are supported." % e)
```

Comment:
Not sure if this should be handled here at all or if we assume they'll always be validated before. Just added just in case.

Reply:
Yeah, this seems like too late for the validation. But if you do keep this, make sure to chain the exception cause explicitly.
```suggestion
        raise AnsibleError("Invalid value for HTTP retry code: %s. Only integer values are supported." % e) from e
```